Fix/factsheet remodel

solar-c opened the pull request:
Closes #311, closes #401, closes #74, closes #36, closes #38, closes #50, closes #82, closes #84, closes #85

4lm commented:
Hi @solar-c,

I'm reviewing your pull request and have a few remarks:

#311: In https://github.com/OpenEnergyPlatform/oeplatform/pull/432/commits/57a35c46a427fbab35fd11965450930b6e49d126 you introduced a method `create_devuser()` in `login/models.py`, which is commented out right in the next commit https://github.com/OpenEnergyPlatform/oeplatform/pull/432/commits/c9ad6edc1d84193319a6c2a88ce73a1a74b731bb. In know that you use this method for easily creating a user while developing locally. IMO it would be OK to introduce this method and remove the multline comment around it.

#401: good!

#74: good! Maybe using a [regex](https://stackoverflow.com/questions/3809401/what-is-a-good-regular-expression-to-match-a-url) or a more complex condition would be a nice to have in the future to exclude false positives. But IMO not a must for now.

#36: This seems to have a bug (maybe I'm wrong). For reproduction, I create a model and framework factsheet and go to "References" ...

... in model factsheet:

- the name "Model usage" is correct, but the description is wrong "Is this framework ...", should be "Is this model ..." - right?

... in framework factsheet:

- the description is correct "Is this framework ...", but the name "Model usage" is wrong, should be "Framework usage" - right?


#38: good!

#50: good!

#82: good!

#84: good!

#85: good!

Bachibouzouk commented on modelview/models.py:
```diff
@@ -674,6 +674,11 @@ def __str__(self):
     )
+    contact_email = EmailField(
+        verbose_name="Contact (e-mail)",
+        help_text="Please provide the mailadress of the contact person.",
```
email address

Bachibouzouk commented on requirements.txt:
```diff
@@ -19,3 +19,4 @@ django-axes==2.3.3
 shapely>=1.5.17
 markdown2
 alembic
+github+https://github.com/solar-c/django-bootstrap3.git
```
Is it a fork? We should maybe host this code also on the OEP github account

solar-c replied:
yes, its a fork. we could also fork it to the OEP github. but lets do it in a new branch/pull request.

Bachibouzouk commented on modelview/static/css/form-style.css:
```diff
@@ -57,3 +57,7 @@ required:0;
     color: red;
 }
+
+.modeledTec th {
```
Update is not a very descriptive commit name ;)

Bachibouzouk commented on login/models.py:
```diff
@@ -51,6 +51,17 @@ def create_superuser(self, name, email, affiliation):
+        user.save(using=self._db)
+        return user"""
+
 
```
I agree with @4lm with leaving the code without the quotes

solar-c commented:
#311: removed multiline comment around new method create_devuser()

#74: maybe if we have some time left :) good remark!

#36: fixed it

Everything should be resolved now
